node/trieIterators/factory/delegatedListHandlerFactory_test.go

```diff
@@ -8,25 +8,32 @@ import (
 	"github.com/multiversx/mx-chain-core-go/core"
 	"github.com/multiversx/mx-chain-go/node/mock"
 	"github.com/multiversx/mx-chain-go/node/trieIterators"
+	trieIteratorsFactory "github.com/multiversx/mx-chain-go/node/trieIterators/factory"
 	"github.com/multiversx/mx-chain-go/testscommon"
 	stateMock "github.com/multiversx/mx-chain-go/testscommon/state"
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func TestCreateDelegatedListHandlerHandler_Disabled(t *testing.T) {
+func TestNewDelegatedListProcessorFactory(t *testing.T) {
+	t.Parallel()
+
+	delegatedListHandlerFactory := trieIteratorsFactory.NewDelegatedListProcessorFactory()
+	require.False(t, delegatedListHandlerFactory.IsInterfaceNil())
+}
+
+func TestDelegatedListProcessorFactory_CreateDelegatedListProcessorHandler_Disabled(t *testing.T) {
```

Comment:
Although I am a fan of this unit test convention naming (with two underscores, one for function to be tested and the other one for the scenario name/state being tested), we do not use it in our projects, therefore is not consistent with the rest of the code.

Each test should contain only one `_`, so this should look like:

```suggestion
func TestDelegatedListProcessorFactory_CreateDelegatedListProcessorHandlerDisabledProcessor(t *testing.T) {
```

! This applies globally for every test